### PR TITLE
Add comparison argument to disassembly output

### PR DESCRIPTION
**Pull request by Saruta**
Output format:
`9       COMPARE_OP              2: '=='`

### CODE REVIEW

File: bytecode.cpp [hunk 1]
```diff
@@ -349,7 +357,9 @@ void bc_disasm(PycRef<PycCode> code, PycModule* mod, int indent)
                 print_const(code->getCellVar(operand), mod);
             } else if (Pyc::IsJumpOffsetArg(opcode)) {
                 fprintf(pyc_output, "%d (to %d)", operand, pos+operand);
-            } else {
+            } else if (Pyc::IsCompareArg(opcode)) {
+		fprintf(pyc_output, "%d: '%s'", operand, cmp_strings[operand]);
```
Comment (zrax) [inline bytecode.cpp]: Please add a range check to avoid accessing `cmp_strings` out of bounds

**Comment by zrax** [comment]
Please use only spaces for indentation (no tabs) to be consistent with the rest of the code base.

**Comment by Saruta** [comment]
Changed, sorry for that.

File: bytecode.cpp [hunk 2]
```diff
@@ -349,6 +357,8 @@ void bc_disasm(PycRef<PycCode> code, PycModule* mod, int indent)
                 print_const(code->getCellVar(operand), mod);
             } else if (Pyc::IsJumpOffsetArg(opcode)) {
                 fprintf(pyc_output, "%d (to %d)", operand, pos+operand);
+            } else if (Pyc::IsCompareArg(opcode)) {
+                fprintf(pyc_output, "%d: '%s'", operand, cmp_strings[operand]);
```
Comment (zrax) [inline bytecode.cpp]: Please add a range check to avoid accessing `cmp_strings` out of bounds

**Pull request merged**